platforms/vitis/cl_firesim/Makefile

```diff
@@ -118,18 +135,26 @@ ipgen_scripts = $(shell find design/ -iname "*.ipgen.tcl")
 	# Move into the simulation or bitstream-build directory
 	cd $(@D)/..; $(VIVADO) -mode batch -source $(buildroot)/scripts/gen_xo.tcl -tclargs $@ $(PROJECT_NAME) $(TARGET) $(DEVICE) $(XSA)
 
+build_strategy_path = $(STRATEGIES_DIR)/strategy_$(STRATEGY).cfg
+
+ifneq ($(shell test -f $(build_strategy_path); echo $$?),0)
+$(error Build strategy config at $(build_strategy_path) does not exist)
+endif
+VPP_CONFIG_OVERLAYS += --config $(build_strategy_path)
+
 # link and package the xclbin
-%/$(PROJECT_NAME).link.xclbin: %/$(PROJECT_NAME).xo
+%/$(PROJECT_NAME).link.xclbin: %/$(PROJECT_NAME).xo $(build_strategy_path)
```

Comment:
Shouldn't you also have a requirement be the `CONFIG_OVERLAY`?

Reply:
Done